controllers/usersignup/usersignup_controller.go

```diff
@@ -512,20 +512,32 @@ func (r *Reconciler) updateUserSignupMetricsByState(oldState string, newState st
 }
 
 func (r *Reconciler) generateCompliantUsername(config toolchainconfig.ToolchainConfig, instance *toolchainv1alpha1.UserSignup) (string, error) {
+	// replaced should now be of len
```

Comment:
Incomplete comment?

Reply:
fixed in [c14186a](https://github.com/codeready-toolchain/host-operator/pull/794/commits/c14186aa4f94ab6a9da5bfdb44a3549c4c17256d)